get_ecr_sts_credentials.sh

```diff
@@ -5,19 +5,39 @@ set -euo pipefail
 #No debug printing!
 set +x
 
-DURATION="${ASSUME_DURATION:=900}"
-MGMT_ACCOUNT="$(cat terraform.tf | grep -i "configured for .* account" | sed 's/.*[^0-9]//')"
+DURATION="${ASSUME_DURATION:=1800}"
+
+# Generate Dev Management Credentials
+DEV_MGMT_ACCOUNT="$(jq -r '."management-dev"' < accounts.json)"
+ROLE_ARN="arn:aws:iam::$DEV_MGMT_ACCOUNT:role/gha_ecr"
+AWS_STS="$(aws sts assume-role --role-arn "$ROLE_ARN" --role-session-name "dev-ci-$(date +%m%d%y%H%M%S)" --duration-seconds ${DURATION})"
+
+DEV_SECRET_ACCESS_KEY="$(echo "$AWS_STS" | jq .Credentials.SecretAccessKey -r)"
+DEV_ACCESS_KEY_ID="$(echo "$AWS_STS" | jq .Credentials.AccessKeyId -r)"
+DEV_SESSION_TOKEN="$(echo "$AWS_STS" | jq .Credentials.SessionToken -r)"
```

Comment:
```suggestion
DEV_SECRET_ACCESS_KEY="$(echo "$AWS_STS" | jq '.Credentials.SecretAccessKey' -r)"
DEV_ACCESS_KEY_ID="$(echo "$AWS_STS" | jq '.Credentials.AccessKeyId' -r)"
DEV_SESSION_TOKEN="$(echo "$AWS_STS" | jq '.Credentials.SessionToken' -r)"
```

From the jq man page [here](https://stedolan.github.io/jq/manual/v1.6/): 

> Note: it is important to mind the shell's quoting rules. As a general rule it's best to always quote (with single-quote characters) the jq program, as too many characters with special meaning to jq are also shell meta-characters.